.zuul.yaml

```diff
@@ -64,6 +64,15 @@
               - ^OWNERS$
               - ^SECURITY_CONTACTS$
               - ^.gitignore$
+    cloud-provider-openstack-e2e-test-csi-cinder:
+      jobs:
+        - cloud-provider-openstack-e2e-test-csi-cinder:
+            irrelevant-files:
+              - ^docs/.*$
+              - ^.*\.md$
+              - ^OWNERS$
+              - ^SECURITY_CONTACTS$
+              - ^.gitignore$
```

Comment:
Is it possible to run e2e test when only the cinder csi related code is changed?

Reply:
done